app/services/preserved_object_handler.rb

```diff
@@ -309,14 +309,14 @@ def increase_version(db_object)
   def confirm_version_on_db_object(db_object, version_symbol)
     results = []
     if incoming_version == db_object.send(version_symbol)
-      results.concat(update_status(db_object, PreservedCopy.statuses[:ok])) if db_object.is_a?(PreservedCopy)
+      results.concat(update_status(db_object, 'ok')) if db_object.is_a?(PreservedCopy)
       results << result_hash(VERSION_MATCHES, db_object.class.name)
     elsif incoming_version > db_object.send(version_symbol)
       # FIXME: this needs to use the same methods as update_version_after_validation
       results.concat(increase_version(db_object))
     else
       # TODO: needs manual intervention until automatic recovery services implemented
-      results.concat(update_status(db_object, PreservedCopy.statuses[:expected_version_not_found_online])) if db_object.is_a?(PreservedCopy)
+      results.concat(update_status(db_object, 'expected_version_not_found_online')) if db_object.is_a?(PreservedCopy)
```

Comment:
I forgot to change this in my PR, but lets change the name of this to `expected_version_not_found_on_storage`

Reply:
@SaravShah: done, pushed it as a follow-on commit since none of the existing commits jumped out at me as the right place to put it.